include/remill/Arch/Runtime/Operators.h

```diff
@@ -560,6 +576,13 @@ ALWAYS_INLINE uint8_t issignaling(float64_t x) {
   return x_nan.exponent == 0x7FFU && !x_nan.is_quiet_nan && x_nan.payload;
 }
 
+ALWAYS_INLINE uint8_t issignaling(float80_t x) {
```

Comment:
Return `bool`. Why not decompose the float 80 directly?? Downcasting will presumably signal if it is a signalling NaN...?

Reply:
Lower down in the code there is ```ALWAYS_INLINE static uint8_t IsSignalingNaN(float80_t x) {
  const nan80_t x_nan = {x};
  return x_nan.exponent == 0x7FFFU && !x_nan.is_quiet_nan && x_nan.payload && x_nan.interger_bit;
}``` which looks more correct.